lang/php7/Makefile

```diff
@@ -572,6 +581,7 @@ $(eval $(call BuildModule,gettext,Gettext,+PACKAGE_php7-mod-gettext:libintl-full
 $(eval $(call BuildModule,gmp,GMP,+PACKAGE_php7-mod-gmp:libgmp))
 $(eval $(call BuildModule,hash,Hash))
 $(eval $(call BuildModule,iconv,iConv,$(ICONV_DEPENDS)))
+$(eval $(call BuildModule,imap,Imap,+PACKAGE_php7-mod-imap:libopenssl))
```

Comment:
Put the dependency on `uw-imap` here instead.

Reply:
how could I put a PKG_BUILD_DEPENDS here ? I tried adding the line with + or without it and is not taken by the build system
there is no actually needed for the package on the system, is needed only for building the module

Reply:
I'm saying do:

```
(eval $(call BuildModule,imap,Imap,+PACKAGE_php7-mod-imap:libopenssl +PACKAGE_php7-mod-imap:uw-imap))
```

Reply:
This is not good, it will want the uw-imap package, but this one is only a dev library and no packaging is made for it